src/components/Footer/Footer.react.js

```diff
@@ -7,7 +7,7 @@ import styled from 'styled-components';
 const FooterContainer = styled.div`
   background: #f6f6f6;
   width: 100%;
-  position: absolute;
+  position: relative;
```

Comment:
Why do you need position now? Also, left and bottom?

Reply:
Removed them